src/chum.cpp

```diff
@@ -167,7 +168,7 @@ void Chum::refreshPackagesFinished()
     package->setPkidLatest(p);
   }
 
-  setStatus(QStringLiteral());
+  setStatus(QLatin1String());
```

Comment:
I wouldn't replace it here. Interesting post in this regard: https://forum.qt.io/topic/78540/qstringliteral-vs-qlatin1string

As we set the value to m_status - which is a qstring, we may create empty qstring straight away and avoid conversion.